WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountPercentageView.swift

```diff
@@ -56,27 +55,33 @@ private extension AddCustomAmountPercentageView {
 private extension AddCustomAmountPercentageView {
     struct PercentageInputField: View {
         @ScaledMetric private var scale: CGFloat = 1.0
+        @FocusState private var focusPercentageInput: Bool
         @Binding var text: String
         var onChangeText: (String) -> (Void)
 
         var body: some View {
-            HStack(spacing: 0) {
+            ZStack {
                 TextField("",
                           text: $text,
                           prompt: Text("0").foregroundColor(Color(.textSubtle))
                 )
                 .onChange(of: text, perform: onChangeText)
                 .focused()
-                .font(.system(size: Layout.percentageFontSize(scale: scale), weight: .bold))
+                .focused($focusPercentageInput)
                 .keyboardType(.decimalPad)
-                .frame(maxWidth: Layout.textFieldMaxWidth)
-                .fixedSize()
+                .opacity(0)
 
-                Text("%")
+                Text(text + "%")
```

Comment:
I'm not sure if the percentage symbol shows differently in RTL locales, but perhaps is worth to cover the case:
```
Text(String.localizedStringWithFormat("%@%%", text))
```

Reply:
Since we need to address it anyway, better to do it with a formatter. A good time will be when we update the calculation approach as discussed yesterday. No need to make a confusing localized string for now; it's been this way for ages without hearing about it anyway.